### ISSUE TITLE
Смещаются элементы в модераторском интерфейсе после отправки комментария

### ISSUE THREAD

**Issue by akmetainfo** [OpenCorpora/opencorpora#647]
Есть у меня пара десятков пулов, в которых осталось только проставить #subst + частично правильно (а все примеры проверены).

Так вот обратил внимание вот на что. Когда добавляешь комментарий, нажимаешь кнопку "отправить комментарий" и ведёшь мышку к выпадающему списку, чтобы выбрать в нём пункт "частично правильно" -- этот выпадающий список дёргается с непредсказуемым результатом.

Бывало уже успел нажать в выпадающем списке "частично правильно" -- тогда результат может перескочить на "нет правильного разбора", либо ты ещё не донёс мышку и клик получается в поле "выбор падежа" (можно нечаянно сменить на другой).

В общем, приходится быть особо внимательным и сконцентрированным, отвлекаться на интерфейс, чтобы он не "убегал" и нечаянно не изменить ответ на неправильный.

Можно ли попробовать убрать это дёргание?

Я смотрел механику процесса. Как только ставишь комментарий -- медленно гаснет (fade) надпись "ваш комментарий добавлен" и как только он полносью исчезает -- тут и происходит сдвиг элементов.

**Comment by grandsbor** [comment]
cc @madfriend

**Comment by grandsbor** [comment]
cc @yourmary

**Comment by madfriend** [comment]
а эта проблема еще вообще воспроизводится? помню, причина вроде была в том, что сверху появляется-исчезает блок с сообщением "ваш комментарий сохранен" (как-то так), я сделал, чтобы он появлялся по-другому

**Comment by akmetainfo** [comment]
Да, вопроизводится. Причина не в "комментарий сохранён", причина чуть другая, хотя и аналогичная. Как только нажимаешь линк "оставить комментарий" - высота строки увеличивается, чтобы хватило места для текстового поля и кнопок.

Элементы модерирования если бы были прибиты valign=top то остались бы на месте, а они центрированы по вертикали.

При нажатии на комментарий происходит первое дёргание, на которое не обращаешь внимания, потому что пишешь текст. 
А когда нажимаешь "сохранить", то высота уменьшается обратно - но не сразу целиком, а (видимо за счёт какой-то задержки в эффектах типа fade - подробно не смотрел) - а в два рывка. 
И вот эти рывки как раз и замечаются, пока курсор идёт в одно место, а в процессе движения цель смещается.
